estimagic/tests/conftest.py

```diff
@@ -1,7 +1,58 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
 import pytest
 
 
 @pytest.fixture(scope="function", autouse=True)
 def fresh_directory(tmpdir):
     """Each test is executed in a fresh directory."""
     tmpdir.chdir()
+
+
+@pytest.fixture
+def example_params():
+    p = (
+        Path(__file__).resolve().parent
+        / "optimization"
+        / "fixtures"
+        / "reparametrize_fixtures.csv"
+    )
+    params = pd.read_csv(p)
+    params.set_index(["category", "subcategory", "name"], inplace=True)
+    for col in ["lower", "internal_lower"]:
+        params[col].fillna(-np.inf, inplace=True)
+    for col in ["upper", "internal_upper"]:
+        params[col].fillna(np.inf, inplace=True)
```

Comment:
Again, if you refactored a function for this, use it here.

Reply:
Done